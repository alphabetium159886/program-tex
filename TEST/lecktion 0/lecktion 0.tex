\documentclass[12pt, a4paper, oneside]{ctexart}
\usepackage{amsmath, amsthm, amssymb, graphicx}
\usepackage[bookmarks=true, colorlinks, citecolor=blue, linkcolor=black]{hyperref}
\usepackage[margin = 25mm]{geometry}
\usepackage{setspace}
\usepackage{listings}
\usepackage{cite}
\usepackage{tabularx}
\usepackage{fancyhdr}
\title{Homework 0:Nobel Laureates Related to Lasers}
\date{\today}
\author{物理2001孙陶庵202011010101\\https://github.com/xingxia1/test1}
\begin{document}
\begin{spacing}{2.0}
\maketitle

\section{123}
一根光纤中的模式频率可以使用波动方程计算：

$\nabla^2\vec{E}-\frac{1}{c^2}\frac{\partial^2\vec{E}}{\partial t^2} = 0$

其中，$\vec{E}$是电场，$c$是真空中的光速，$\nabla^2$是拉普拉斯算子。

假设电场可分为径向和方位角两个部分，则我们可以写成：

$\vec{E}(r,\theta,z,t)=\vec{E}(r)e^{-i\beta z}e^{i\omega t}\vec{u}(\theta)$

其中$r$是距离光纤轴线的径向距离，$\theta$是方位角度数，$z$是沿着光纤的距离，$\beta$ 是传播常数, $\omega $ 是角频率, $\vec {u } (\theta ) $ 是方位角单位矢量。

将这个表达式代入波动方程并简化后得到：

$
    \frac {1}{r}\frac {\partial }{\partial r}\left(r \frac {\partial \vec {E}}{\partial r }\right)+(\beta ^ 2-\frac {\omega ^ 2}{c ^ 2} n ^ 2 (r)) \ vec { E}=0
$

其中$n(r)$表示在径向距离$r $处的折射率。

该方程可以通过适当的边界条件求解，例如，在芯层-包层边界处电场及其导数的连续性。

模式频率与其角频率$\omega$有关：

$
    \omega =\frac {2\pi c}{\lambda}
$

其中，$\lambda $是光在光纤中的波长。

因此，模式频率可以表示为：

$
    f=\frac {\omega }{2\pi}=\frac {c}{\lambda}
$

其中$f$是模式的频率。

总之，在光纤中的模式频率与其波长有关，并且可以使用波动方程和适当的边界条件进行计算。该频率由光纤的折射率分布以及芯区域大小和形状决定。\\

光纤中导模的传播常数$\beta$由以下公式给出：

$\beta = k_0 n_{eff}$

其中$k_0$是光在真空中的波数，$n_{eff}$是模式的有效折射率。有效折射率与芯层和包层的折射率有关：

$n_{eff} = \sqrt{\frac{\int_{core} n^2(r) \, dA}{\int_{core} \, dA}}$

其中$n(r)$是芯内折射率的径向分布，积分取自芯部横截面积。对于半径为$a$、折射率为$n_1$、被折射率为$n_2$所包围的阶跃型光纤，其折射率径向分布如下：

$n(r) = \begin{cases} n_1, & r \le a \\ n_2, & r > a \end{cases}$

将此表达式代入计算$n_{eff}$ 的公式并求解积分得到：

$n_{eff} = \sqrt{n_1^2 \frac{2\pi}{\beta^2}\int_0^a r \sqrt{\beta^2 - k_0^2 n_1^2 r^2} \, dr + n_2^2 \frac{2\pi}{\beta^2}\int_a^{\infty} r \sqrt{\beta^2 - k_0^2 n_2^2 r^2} \, dr}$

为了找到支持导模的$\beta$值范围，我们需要在芯层-包层界面处施加边界条件。在该界面上，电场和磁场的切向分量必须连续。这些边界条件导致以下方程式来计算$\beta$：

$\tan\left(\frac{\beta a}{2}\right) = \sqrt{\frac{n_{eff}^2 - n_2^2}{n_1^2 - n_{eff}^2}}$

此方程将传播常数$\beta$与有效折射率$n_{eff}$和芯层、包层的折射率$n_1$和$n_2$相关联。仅当根号下的量为正时，它才有$\beta$ 的解，这导致存在导模的以下条件：

$k_0 n_2 < \beta < k_0 n_1$

此条件定义了支持光纤中导模传播常数范围。
\end{spacing}{}

\bibliographystyle{IEEEtran}
\bibliography{lecktion0}

\end{document}