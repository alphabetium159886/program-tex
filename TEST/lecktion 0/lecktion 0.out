\BOOKMARK [1][-]{section.1}{\376\377\0001\0002\0003}{}% 1
